### ISSUE TITLE
Why most useful features got removed in the current branch?

### ISSUE THREAD

**Issue by rsax02** [TheSharks/WildBeast#664]
Some useful features like per-user permissions level (++setlevel), per-guild settings (++settings) and most bot configuration available in `.env` got deleted in the current branch. Will they be added again?

Most of these features got removed in 34ad4c93d7e09589ef84cc8fecd784ae04a4d158.

Thanks in advance.

**Comment by Dougley** [comment]
We found that our classic level system introduced a lot of friction, especially when people don't understand it. We've replaced this with the prerequisites system which is more Discord-native.

Per-guild customization is going to return at some point, it's on the roadmap.

As for the other differences with v6:
- i18n (translation support) is going to return in the future
- Multiple encoder support (eg. Lavalink and ffmpeg) is removed. ffmpeg was vastly inferior to Lavalink, and people complained it harmed the quality too badly
- Multiple database support has evolved into the databases our SQL query builder, [Knex](http://knexjs.org/#Installation-node), supports
- Bezerk is removed
- Elasticsearch got removed in favor of Prometheus
- Rancher scaling got removed in favor of Kubernetes scaling

The documentation is still pending updates to reflect all of these changes accordingly, but that's being worked on at the moment.
